testsuite/tests/tool-toplevel/reloc.ml

```diff
@@ -0,0 +1,9 @@
+(* TEST
+   include ocamlcommon
+   * toplevel
```

Comment:
Could you turn this into an expect test? It seems to me that we are generally turning tests into expectation tests

Reply:
I'm not sure how this work. I'll have a look

Reply:
This is now done